.env.test

```diff
@@ -1 +1 @@
-DATABASE_CONN="mongodb://127.0.0.1:27017/music-library-api-test"
+DATABASE_CONN=mongodb+srv://cai:linfield@cluster0-q5mno.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
⚠️SECURITY ALERT! ⚠️ 

Your database string, and password are on github. Your best course of action is to delete the database user and create a new string.